roles/openafs_devel/tasks/install/CentOS-7.yaml

```diff
@@ -20,7 +20,7 @@
       - gcc
       - git
       - glibc-devel
-      - "kernel-devel-uname-r == {{ ansible_kernel }}"
```

Comment:
This not a error, it is intentional, so the kernel devel that matches the running kernel will be installed.

Reply:
Manually upgrading the kernel then re-running the molecule converge worked.